### ISSUE TITLE
Vertical scrolling throws an error on blog with single post

### ISSUE THREAD

**Issue by bravo-kernel** [narative/gatsby-theme-novela#49]
To reproduce:

```bash
gatsby new novela-tester https://github.com/narative/gatsby-starter-novela
cd novela-tester
npm install
```

Now:

1. browse to http://localhost:8000/
2. do not scroll the page
3. open chrome inspector, console
4. browse to http://localhost:8000/my-first-post-using-novela-by-narative
5. browse back to http://localhost:8000/
6. scroll page

**Check console:**

```
Warning: Can't perform a React state update on an unmounted component. 
This is a no-op, but it indicates a memory leak in your application.
To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
```

**Comment by brotzky** [comment]
fixed in v0.2.1

**Comment by bravo-kernel** [comment]
Confirmed as fixed, thank you kindly 💯
